### PR TITLE
fixes #2528 iliac vein smooth muscle tissue

### LINKED ISSUE #2528: iliac vein smooth muscle tissue
**Preferred term label:** iliac vein smooth muscle tissue
**Synonyms** muscle of iliac vein
**Definition** A portion of smooth muscle tissue that is part of an iliac vein
**Parent term** 
iliac vein UBERON:0005610
venous system smooth muscle UBERON:0004696
**Your nano-attribution (ORCID)** 0000-0002-0819-0473

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #2528 iliac vein smooth muscle tissue

### CODE REVIEW

**Comment by bvarner-ebi** [comment]
@paulacalipho, I now see this is failing a QC check because a dbxref is missing.

`ERROR: definition-lacks-dbxref: UBERON:8480000`

If you have a paper that gives evidence for this term, it should be added as an annotation to the definition.

**Comment by shawntanzk** [comment]
It's just reordering. When multiple PRs are simultaneously done (which happens a lot for me) it might not go in in the right order. When you save something in protege, it puts terms in the right order, hence you seeing the rearrangement of terms. As long as whatever is removed is added back, I wouldn't worry about it

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
Add dbxref to definition

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224180,6 +224180,16 @@ is_a: UBERON:0000029 ! lymph node
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
 property_value: dcterms-date 2022-05-26T11:59:51Z xsd:dateTime
 
+[Term]
+id: UBERON:8480000
+name: iliac vein smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of an iliac vein." []
+synonym: "muscle of iliac vein" BROAD []
+is_a: UBERON:0004696 ! venous system smooth muscle
```
Comment (shawntanzk) [inline src/ontology/uberon-edit.obo:224188]: I would instead consider equivalent class of:
'smooth muscle tissue' and ('part of' some 'iliac vein')

The reasoner should classify it a venous system smooth muscle 
It's usually good practice to let reasoners do the work to ensure that you dont miss out classifications, fall into some unexpected hidden GCI, etc. furthermore axioms should reflect the definition in logical axioms as far as possible.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224180,6 +224180,16 @@ is_a: UBERON:0000029 ! lymph node
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
 property_value: dcterms-date 2022-05-26T11:59:51Z xsd:dateTime
 
+[Term]
+id: UBERON:8480000
+name: iliac vein smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of an iliac vein." []
+synonym: "muscle of iliac vein" BROAD []
```
Comment (shawntanzk) [inline src/ontology/uberon-edit.obo:224187]: synonyms should ideally come with reference too (I know @bvarner-ebi requested dbxref for def already, but synonyms should ideally have them too)

**Review by shawntanzk — CHANGES_REQUESTED** [review]
see in line

**Comment by paulacalipho** [comment]
I understand that I must do the following changes:
1) change is_a: UBERON:0004696 ! venous system smooth muscle to is_a: UBERON:0001135 ! smooth muscle tissue
2) add dbxref to definition
3) add dbxref to synonym

Questions:
I do not have dbxref for definition or synonym (I "automatically generated"). What I must do in this case?
Should I close this pull request and start from the beginning?

**Comment by shawntanzk** [comment]
> change is_a: UBERON:0004696 ! venous system smooth muscle to is_a: UBERON:0001135 ! smooth muscle tissue

Not exactly - but basically you should change 
`is_a: UBERON:0004696 ! venous system smooth muscle` 
to
```
intersection_of: UBERON:0001135 ! smooth muscle tissue
intersection_of: part_of UBERON:0005610! iliac vein
```
(probably easier done in protege, and do run the ELK reasoner to ensure that nothing unexpected pops up) 

> I do not have dbxref for definition or synonym (I "automatically generated"). What I must do in this case?

for definition, add your orcid as a dbxref works (make sure you use https://)
for synonym, you should find a paper that uses and at the PMID/DOI of that paper (less crucial than the definition)

> Should I close this pull request and start from the beginning?

There isn't a need to, just make corrections on the branch and push the changes to this PR :)

**Comment by paulacalipho** [comment]
Thanks @shawntanzk for your replay. I made the changes and pushed the changes to this PR. Not sure I did correctly (sorry)
Also thanks to @bvarner-ebi for his great help on Friday

File: src/ontology/uberon-edit.obo [hunk 3]
```diff
@@ -224180,6 +224180,16 @@ is_a: UBERON:0000029 ! lymph node
...
 property_value: dcterms-date 2022-05-26T11:59:51Z xsd:dateTime
 
+[Term]
+id: UBERON:8480000
+name: iliac vein smooth muscle tissue
+def: "A portion of smooth muscle tissue that is part of an iliac vein." [https://orcid.org/0000-0002-0819-0473]
+intersection_of: UBERON:0001135 ! smooth muscle tissue
+intersection_of: part_of UBERON:0005610 ! iliac vein
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0005610 ! iliac vein
```
Comment (shawntanzk) [inline src/ontology/uberon-edit.obo:224190]: This subclass axiom should be removed as it is already in the equivalent class axiom

Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:224190]: should I remove "relationship: part_of UBERON:0005610 ! iliac vein" and do the PR again, right?

**Review by bvarner-ebi — APPROVED** [review]

**Review by shawntanzk — APPROVED** [review]

**Pull request merged**